api/v1beta1/auth_config_types.go

```diff
@@ -698,7 +698,21 @@ type AuthConfig struct {
 type AuthConfigList struct {
 	metav1.TypeMeta `json:",inline"`
 	metav1.ListMeta `json:"metadata,omitempty"`
-	Items           []AuthConfig `json:"items"`
+	Items           AuthConfigSlice `json:"items"`
+}
+
+type AuthConfigSlice []AuthConfig
+
+func (s AuthConfigSlice) Len() int {
+	return len(s)
+}
+
+func (s AuthConfigSlice) Less(i, j int) bool {
```

Comment:
Ok, so `sort.Sort` will sort the `Items` slice based on `CreationTimestamp` comparison :+1:

Reply:
Correct